### DISCUSSION TITLE
warning on readme

### DISCUSSION THREAD

**Discussion by infa-ddeore** [acorn-io/runtime#566] [General]
below warning is given on README page, how can a software use cause this?

```
Caution: Consuming this project can expose you to chemicals, which are known to the State of California to cause cancer and birth defects or other reproductive harm.
```

**Comment by vincent99** [comment]
It's a joke.  These signs are somewhat famously all over everything in California…

https://en.wikipedia.org/wiki/1986_California_Proposition_65#Controversy_and_claimed_abuse

**Comment by infa-ddeore** [comment]
i see :D
